event-sourcing/event-publisher/event-publisher-timer/src/main/java/uk/gov/justice/services/eventsourcing/prepublish/PrePublishTimerConfig.java

```diff
@@ -17,6 +17,10 @@
     @GlobalValue(key = "pre.publish.timer.interval.milliseconds", defaultValue = "500")
     private String timerIntervalMilliseconds;
 
+    @Inject
+    @GlobalValue(key = "pre.publish.timer.cutoff.milliseconds", defaultValue = "500")
```

Comment:
The configuration value 'pre.publish.timer.cutoff.milliseconds' can be removed

Reply:
I have removed the maxEventsPublishedPerIteration value